docs/static/persistent-queues.asciidoc

```diff
@@ -19,10 +19,14 @@ Instead of deploying and managing a message broker, such as Redis, RabbitMQ, or
 Apache Kafka, to facilitate a buffered publish-subscriber model, you can enable
 persistent queues to buffer events on disk and remove the message broker.
 
-In summary, the two benefits of enabling persistent queues are as follows:
+In summary, the benefits of enabling persistent queues are as follows:
 
 * Provides protection from in-flight message loss when the Logstash process is abnormally terminated.
 * Absorbs bursts of events without needing an external buffering mechanism like Redis or Apache Kafka.
+* Provides an at-least-once message delivery guarantee. If Logstash is restarted while events are
```

Comment:
@dedemorton ++++

One minor comment "shutdown or abnormally terminated" to me feels like we're equivocating "shut down" and "abnormally terminated" . Thinking outloud, maybe minor rephrase to say "against message loss for both normal shutdown as well as abnormal termination" ?